skfair/metrics/fairness_report.py

```diff
@@ -0,0 +1,90 @@
+from collections import defaultdict
+
+from sklearn.metrics import confusion_matrix
+import numpy as np
+
+
+def true_false_positive_negative(conf_matrix):
+    # conf matrix: rows are true, cols are pred
+    if conf_matrix.shape == (2, 2):  # binary case
+        tn, fp, fn, tp = conf_matrix.ravel()
+        return tn, fp, fn, tp
+    tp = 0
+    fp = 0
+    fn = 0
+    tn = 0
+    # A second check on the below is appreciated.
+    for i in range(conf_matrix.shape[0]):
+        tp += conf_matrix[i, i]
+        fn += conf_matrix[i, :].sum() - conf_matrix[i, i]
+        fp += conf_matrix[:, i].sum() - conf_matrix[i, i]
+        tn += (conf_matrix[:i-1, :i-1].sum() + conf_matrix[:i-1, i+1:].sum() +
+               conf_matrix[i+1:, :i-1].sum() + conf_matrix[i+1:i+1:].sum())
+    return tn, fp, fn, tp
+
+
+def classification_fairness_report(y_true, y_pred, groups, group_names=None,
+                                   labels=None, output_dict=False):
```

Comment:
Would it not be preferable here to have `output` instead of `output_dict`? 

This way the user can specify `output="dict"`, `output="text"` or `output="pandas"`.

Reply:
output_dict is the sklearn parameter, your suggestions sounds better, let's change to that.